src/dask_awkward/lib/io/columnar.py

```diff
@@ -44,6 +44,8 @@ def attrs(self) -> dict | None: ...
 
     def project_columns(self: T, columns: frozenset[str]) -> T: ...
 
+    def project_manually(self: T, columns: frozenset[str]) -> ImplementsIOFunction: ...
```

Comment:
This is fully specified in the implementation of ColumnProjectionMixin, right? So does the protocol definition here serve any purpose?

Reply:
I may be able to remove it without mypy complaining, I'll check. Apart from that, I was adding it here as other ones were defined for this protocol aswell, e.g. `project_columns`.